.gitignore

```diff
@@ -1,8 +1,59 @@
-__pycache__
-*.egg-info
-outputs/
+# OS-generated
+# ------------
+.DS_Store*
+[Tt]humbs.db
+[Dd]esktop.ini
+
+# Programming - general
+*.log
+
+
+# =========================================================================== #
+# 	Python-related
+# =========================================================================== #
+# src: https://github.com/github/gitignore/blob/master/Python.gitignore
+
+# JetBrains PyCharm / Rider
+.idea/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
 .env_updated
 condaenv.*.requirements.txt
-models/ldm/stable-diffusion-v1/model.ckpt
-src/gfpgan/experiments/pretrained_models/*.pth
-src/realesrgran/experiments/pretrained_models/*.pth
```

Comment:
These need to be ignored too

Reply:
- the `*.ckpt` model is excluded with a separate `/models/ldm/.gitignore` 
- since `/src` dir is only used by pip to install from git in-place, I excluded it entirely.

Are you sure these rules do need to be added, too?